### PR TITLE
Wrapper for MemoryStream

### PR DESCRIPTION
**Pull request by hougaard**
Wrapper for MemoryStream - Can also be used for Text In/Out

### CODE REVIEW

**Comment by BardurKnudsen** [comment]
Code looks good. Is it to replace the usage of BLOB fields, e.g. TempBlob?  What are the usage scenarios?

**Comment by hougaard** [comment]
This is certainly a replacement for TempBlob, but also for use with the HttpClient in cases where you're downloading binaries etc..
This is one DotNet type I use a lot.

**Comment by hougaard** [comment]
And it could properly also be a workaround for the missing encoding on HttpClient :) (Because you can put encodings on in and out streams.

File: Memory Stream.txt [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+  CODE
+  {
+    VAR
+      MemoryStream@1160530000 : DotNet "'mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089'.System.IO.MemoryStream";
+      StreamWriter@1160530001 : DotNet "'mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089'.System.IO.StreamWriter";
+      StreamReader@1160530002 : DotNet "'mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089'.System.IO.StreamReader";
+
+    PROCEDURE Create@1160530001(InitialSize@1160530000 : Integer);
+    BEGIN
+      MemoryStream := MemoryStream.MemoryStream(InitialSize);
```
Comment (Gallimathias) [inline Memory Stream.txt]: Practical would be the implementation of the empty constructor too.

Comment (Gallimathias) [reply Memory Stream.txt]: Besides, reading and writing would be interesting and I think dispose is very important.

**Comment by JesperSchulz** [comment]
If all goes according to plan, I will take a look tomorrow and try to get the code into our BaseApp before end of week. Stay tuned!

**Comment by gunnargestsson** [comment]
I Don't think this should replace TempBlob but would rather add functions like CopyToTempBlob(VAR TempBlob: Record TempBlob) and InitializeFromTempBlob(TempBlob: Record TempBlob) to this library

**Comment by JesperSchulz** [comment]
Code submitted through pull request #9. Closing this pull request.

**Comment by JesperSchulz** [comment]
Code will be released through November Update. Thank you for your contribution!

**Pull request closed without merging**